profitbricks/pom.xml

```diff
@@ -36,7 +36,11 @@
         <test.profitbricks.identity>FIXME</test.profitbricks.identity>
         <test.profitbricks.credential>FIXME</test.profitbricks.credential>
         <test.profitbricks.api-version>1.3</test.profitbricks.api-version>
-        <test.profitbricks.template />
+        <test.profitbricks.template>
+            locationId=45a8667e-edfe-4f64-8a4d-79d53c8fec1f,
+            imageNameMatches=jclouds-ubuntu14.10-template,
+            loginUser=root:asdfghjk
+        </test.profitbricks.template>
```

Comment:
Is this a public template anyone will be able to use, or is it a custom one in your account? If it is a private one and the ones provided by default by Profitbricks don't provide remote access, I'd add a README with the instructions to build a  *jclouds friendly* image and how to run the live tests using it.